snoop/data/migrations/0040_thumbnail.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 3.1.4 on 2021-04-15 11:23
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('data', '0039_auto_20210204_2125'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Thumbnail',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('blob', models.OneToOneField(on_delete=django.db.models.deletion.CASCADE, to='data.blob')),
+                ('original', models.OneToOneField(on_delete=django.db.models.deletion.CASCADE, to='data.digest')),
```

Comment:
Why do we need the `original` here?

Just run the service on the `digest.blob` object and ignore the `original` stuff.